src/StackParameters.hs

```diff
@@ -199,6 +199,7 @@ stackOutputName = StackOutput <$>
     )
 
 getParameters :: MonadThrow m => Environments -> AWSAccountID -> m Parameters
+getParameters envs _ | HashMap.null envs = pure mempty
```

Comment:
If your going to use pattern guard, why not use `otherwise` instead of declaring the function and values again:
```haskell
getParameters envs acc
  | HashMap.null envs = pure mempty
  | otherwise         = case HashMap.lookup acc envs of
    Nothing -> throwM $ EnvironmentNotFound acc
    Just ps -> return ps
```